tests/test_reverse_string.py

```diff
@@ -0,0 +1,26 @@
+from unittest import TestCase, main
+from task_2.reverse_string import reverse_string, NonStringInputError
```

Comment:
1. It's a good practice to have an empty line between built-in packages imports and your files imports

2.  I the task_2 file, in init file, please import your function and error like
```
from .reverse_string import reverse_string, NonStringInputError
```
So here you would be able to import your code directly
```
from task_2 import reverse_string, NonStringInputError
```

Reply:
Thanks for tips, I've already employed them.